frontend/src/locales/scripts/en.json5

```
@@ -621,9 +621,9 @@
         countMore: "See over {localeCount} images found for {query}.",
       },
       audio: {
-        zero: "No audio found for {query}.",
-        count: "See {localeCount} audio found for {query}.",
-        countMore: "See over {localeCount} audio found for {query}.",
+        zero: "No audio track found for {query}.",
+        count: "See {localeCount} audio track found for {query}.",
+        countMore: "See over {localeCount} audio track found for {query}.",
```

Please use the plural form.
```suggestion
        count: "See {localeCount} audio tracks found for {query}.",
        countMore: "See over {localeCount} audio tracks found for {query}.",
```